css/lists.css

```diff
@@ -24,3 +24,35 @@
 .spaced-list li {
 	margin: 1rem;
 }
+
+.dropdown li {
+	width: 1rem !important;
```

Comment:
Try not to use important, See if you can achieve the outcome by changing the specificity